lib/galaxy/selenium/navigation.yml

```
@@ -165,37 +167,46 @@ history_panel:
       list_items: '.dataset-collection-panel .list-items .list-item'
 
   selectors:
+    beta: 'section.history'
     _: '#current-history-panel'
     search: '#current-history-panel input.search-query'
-    refresh_button: '#history-refresh-button'
+    refresh_button: '.history-refresh-button'
     name: '.title .name'
+    name_beta: '.history-title span:last-child'
     name_edit_input: '.name input'
+    name_edit_input_beta: '.history-title input'
     contents: '#current-history-panel .list-items div.history-content'
 
     empty_message: '.empty-message'
     size: '.history-size'
     tag_icon: '.actions .history-tag-btn'
-    tag_area: '.controls .tags-display'
-    tag_area_input: '.controls .tags-display .tags-input input'
+    tag_area: '.history-details .tags-display'
+    tag_area_input: '.history-details .tags-display .tags-input input'
     tag_close_btn: '.tags-display .ti-icon-close'
     tags: 'li.ti-tag.ti-valid .tag-name'
     annotation_icon: '.actions .history-annotate-btn'
-    annotation_area: '.controls .annotation-display'
-    annotation_editable_text: '.controls .annotation-display .annotation.editable-text'
-    annotation_edit: '.controls .annotation-display .annotation textarea'
-    annotation_done: '.controls .annotation-display .annotation button'
+    annotation_area: '.history-details .history-annotation'
+    annotation_editable_text: '.history-details .history-annotation .annotation.editable-text'
+    annotation_editable_text_beta: '.history-details .history-annotation.clickToEdit'
+    annotation_edit: '.history-details .history-annotation .annotation textarea'
+    annotation_edit_beta: '.history-details .history-annotation textarea'
+    annotation_done: '.history-details .history-annotation .annotation button'
 
     options_button: '#history-options-button'
     options_button_icon: '#history-options-button span.fa-cog'
+    options_button_icon_beta: 'section.history-details .priority-menu-expand'
```

```suggestion
    options_button_icon_beta: 'section .history-details .priority-menu-expand'
```

This is probably a typo, because I cannot find `section.history-details .priority-menu-expand` in DOM, while `section .history-details .priority-menu-expand` is the "three dot" button, we are looking for.

but in general we can unify this button. Is the separation truly needed here? we can add `id=history-options-button span` in [here](https://github.com/jmchilton/galaxy/blob/9107e01362225d63f209f8127ecc84af5df768b3/client/src/components/PriorityMenu/Menu.vue#L17) and remove span from `options_button_icon: '#history-options-button span.fa-cog'`

Alternatively we can add smth like `.menu-expand-button` in both

---

I went ahead and tried to make this more expansive change you recommended. Lets see what the test results look like.